File: exo3/exo3.java
//comptage du nombre de caractere 
//author sebastien
//beWeb

import java.util.*;
//importation de la bibliotheque java

public class exo3{
//appel du fichier compile
	public static void main(String[] args){
	//on recupere les parametre passer dans le terminal
	//on stocke le 1er parametre dans le string word
		String word = args[0];
	//on stocke le 1er parametre dans le string word
		String carac = args[1];
		int resultat = 0;
		int i = 0;
	//on boucle tanque i est inferieur a la longueur de word
		while(i < word.length()){
		//word.charAt(i) => on recupere le caractere a l'index i de word
		//carac.charAt(0) => on recupere le caractere a l'index 0 de carac
			if(word.charAt(i) == carac.charAt(0)){
				resultat++;
			}
		i++;
		}
	System.out.println("le nombre de caractereRecherche est : "+resultat);
	}

}

File: exo3/exo3.py
#comptage du nombre de caractere 
#author sebastien
#beWeb

#importer le module sys
import sys

def countLetter(word,letter):

    resultat = 0
    i = 0
    #Savoir le nombre d element dans la liste d arguments
    if len(sys.argv) == 0:
        #sys.argv permet de manipuler les arguments [word] et [letter] dans la ligne de commande
        sys.argv[1:word]
        sys.argv[2:letter]


    #si la lettre a l'index i est egale a la lettre de la variable letter
    while i < len(word) :
        #si a l'index ou je me trouve c'est bien un "n" j'incremente de 1
	    if word[i] == letter :
		    resultat += 1
        #et je passe a l'index suivant
	    i += 1

    print('le nombre de ' + str(letter)+' est : ' + str(resultat))
#on recupere les parametre passer dans le terminal
countLetter(sys.argv[1],sys.argv[2])
